.stylelintrc

```diff
@@ -24,6 +24,7 @@
     ],
     "selector-max-id": 0,
     "sh-waqar/declaration-use-variable": "/color/",
-    "string-quotes": "double"
+    "string-quotes": "double",
+    "value-list-comma-newline-after": null
```

Comment:
I added this so that stylelint wouldn't complain about comma-sep font lists (see `index.scss`)